packages/main/src/MultiComboBoxPopover.hbs

```diff
@@ -37,30 +37,73 @@
 						aria-autocomplete="both"
 						aria-labelledby="{{_id}}-hiddenText-nMore"
 						aria-describedby="{{_id}}-valueStateDesc"
-					/>
+						/>
+					</div>
+					<ui5-togglebutton
+						slot="header"
+						class="ui5-multi-combobox-toggle-button"
+						icon="multiselect-all"
+						design="Transparent"
+						?pressed={{_showAllItemsButtonPressed}}
+						?disabled={{allItemsSelected}}
+						@click="{{filterSelectedItems}}"
+					></ui5-togglebutton>
 				</div>
-				<ui5-togglebutton
-					slot="header"
-					class="ui5-multi-combobox-toggle-button"
-					icon="multiselect-all"
-					design="Transparent"
-					?pressed={{_showAllItemsButtonPressed}}
-					?disabled={{allItemsSelected}}
-					@click="{{filterSelectedItems}}"
-				></ui5-togglebutton>
+			{{#if hasValueStateMessage}}
+				<div class="{{classes.popoverValueState}}" style="{{styles.popoverValueStateMessage}}">
+					{{> valueStateMessage}}
+				</div>
+			{{/if}}
+			</div>
 		</div>
-	</div>
+	{{/if}}
+
+	{{#unless _isPhone}}
+		{{#if hasValueStateMessage}}
+			<div slot="header" class="ui5-responsive-popover-header {{classes.popoverValueState}}" style={{styles.popoverValueStateMessage}}>
+				{{> valueStateMessage}}
+			</div>
+		{{/if}}
+	{{/unless}}
 
 	<ui5-list separators="None" mode="MultiSelect" class="ui5-multi-combobox-all-items-list">
 		{{#each _filteredItems}}
 				<ui5-li type="Active" ?selected={{this.selected}} data-ui5-token-id="{{this._id}}">{{this.text}}</ui5-li>
 		{{/each}}
 	</ui5-list>
 
-	<div slot="footer" class="ui5-responsive-popover-footer">
-		<ui5-button
-			design="Transparent"
-			@click="{{togglePopover}}"
-		>OK</ui5-button>
-	</div>
+	{{#if _isPhone}}
+		<div slot="footer" class="ui5-responsive-popover-footer">
+			<ui5-button
+				design="Transparent"
+				@click="{{togglePopover}}"
+			>OK</ui5-button>
```

Comment:
Doen't this OK need to be translatable?

Reply:
I left it the way it was, but also in my opinion it should be translated, therefore I am going to change it.